packages/rn-tester/js/examples/Modal/ModalPresentation.js

```diff
@@ -199,8 +199,8 @@ function ModalPresentation() {
           <RNTOption
             key="onDismiss"
             style={styles.option}
-            label="onDismiss ⚫️"
-            disabled={Platform.OS !== 'ios'}
+            label="onDismiss"
+            multiSelect={true}
```

Comment:
why multiselect must be set to `true`?

Reply:
It's to enable RNTOption
https://github.com/facebook/react-native/blob/b507e118678d5851555f664585bb239f6dcd6022/packages/rn-tester/js/components/RNTOption.js#L40-L44

I see that `onShow` passes `multiSelect={true}` instead of `disable={false}`, so I follow it. 
https://github.com/facebook/react-native/blob/b507e118678d5851555f664585bb239f6dcd6022/packages/rn-tester/js/examples/Modal/ModalPresentation.js#L186-L190

Do you think I should change it to `disable={false}`?

Reply:
yeah, probably it is more clear and accurate. I think that `multiSelect` is the API that allows you to select multiple choices (e.g.: "Which ice cream flavour do you prefer?" you can reply with "Chocolate", "cream" and "vanilla" and leave "strawberry" out) but in our case is is true/false and it does not make sense to select both.
WDYT?

Reply:
> I think that multiSelect is the API that allows you to select multiple choices

This is what I thought when saw it the first time and I totally agree using `disabled` is make more sense. Updated the code.